### DISCUSSION TITLE
sudo: ./tailwindcss: command not found

### DISCUSSION THREAD

**Discussion by babur001** [tailwindlabs/tailwindcss#10372] [Help] [answered]
**What version of Tailwind CSS are you using?**
3@latest

**What build tool (or framework if it abstracts the build tool) are you using?**
CRA

**What version of Node.js are you using?**
v16.19.0

---

Installed and moved executable in root folder (see image below). Running `sudo ./tailwindcss` is giving an error `sudo: ./tailwindcss: command not found`, why is that happening? Tried terminals other than `zsh` still not getting any result.

Thanks in advance!

 
<img width="1280" alt="Screen Shot 2023-01-20 at 20 01 26" src="https://user-images.githubusercontent.com/73885199/213730204-b7243d5a-3f7a-4e76-ad5e-f4acfce936ef.png">

**Comment by babur001** [comment]
Using [tailwindcss-macos-x64](https://github.com/tailwindlabs/tailwindcss/releases/download/v3.2.4/tailwindcss-macos-x64)

**Comment by adamwathan** [comment, marked as answer]
Hey! Have you given it the correct permissions?

```sh
chmod +x tailwindcss
```

**Reply by hristian-carabulea** [reply]
curl -sLO https://github.com/tailwindlabs/tailwindcss/releases/latest/download/tailwindcss-linux64
mv tailwindcss-linux64 tailwindcss
chmod +x tailwindcss
./tailwindcss init
./tailwindcss: line 1: Not: command not found

lsb_release -a
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.1 LTS
Release:	22.04
Codename:	jammy

uname -a
Linux 5.15.0-58-generic #64-Ubuntu SMP Thu Jan 5 11:43:13 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

I cannot make tailwindcss work.

**Comment by babur001** [comment]
Yes, I see, it works! Thanks
